app/controllers/ems_container_controller.rb

```diff
@@ -32,6 +32,11 @@ def ems_container_form_fields
     ems_form_fields
   end
 
+  def new
+    super
+    @ems_types -= [["Atomic", "atomic"], ["Atomic Enterprise", "atomic_enterprise"]]
```

Comment:
@durandom yes... it's just very much counter-intuitive :smile:  ...why `@ems_types` in this class should be the excluded ones? (I know that this belongs to the existing code... I just find it very, very, confusing).